artifacts_test.py

```diff
@@ -371,6 +396,10 @@ def test_scylla_service(self):
                                                             new_row_expected=True,
                                                             backend=backend)
 
+        if backend == 'docker':
```

Comment:
I'm not sure this related to the artifact test specifically, we might need to create a total new test, that would trigger regularly (something like the operator functional-tests)
 
we had a tests like that in the past, checking the download of the releases, @roydahan do we still run them ?

Reply:
@fgelcer you are o.k. with this ? i.e. that might break the artifact_docker on multiple branches. regardless of the artifact under test.

Reply:
I don't think we have them or still running them.

Reply:
AFAICT there are no such artifacts for docker backend...
and in case we want to add such (and maybe we should), in such task, every detail of it, including checking the version, should be done in it, so i'm not afraid of it breaking at this point